packages/nuejs.org/docs/hello-world.md

```diff
@@ -140,7 +141,7 @@ We start by creating a [custom layout file](custom-layouts.html) called `layout.
 
 
 ### Add shared data
-Then we add a data file called `site.yaml` with all the site-wide data to fill in our personal information and other basic data being used on our header and footer:
+We then add a data file named site.yaml, containing all the site-wide data, to populate our personal information and other essential details used in the header and footer:
```

Comment:
```suggestion
We then add a data file named `site.yaml`, containing all the site-wide data, to populate our personal information and other essential details used in the header and footer:
```

Reply:
Thanks for spotting the missing back ticks between site.yaml.